### ISSUE TITLE
How can ProtoSchool best meet the needs of the libp2p team? 

### ISSUE THREAD

**Issue by terichadbourne** [libp2p/libp2p#70]
As we build the roadmap for [ProtoSchool](https://proto.school), we'd like to take into account the priorities of the libp2p team and plan for some tutorial content that best highlights your most common or most prioritized use cases or features. 

For a sense of what ProtoSchool is capable of, please take a look at the [existing tutorials](https://proto.school/#/tutorials), which run in-browser and (with the exception of the first tutorial on the list) offer coding challenges following the introduction of various content step-by-step. Beginner-friendliness is a major priority for this project, and as we consider requests from project teams we will also focus on ensuring that appropriate scaffolding exists to get users to the point where they can successfully approach those proposed topics.

Could you please take a look at your project roadmap and help me understand what ProtoSchool tutorial content might most help you achieve your goals for 2019 and 2020? 

Do you have upcoming events where you hope to offer workshops? If you could envision that content fitting with the ProtoSchool tutorial format, please be sure to include these ideas and share the relevant event dates.

cc @mikeal

**Comment by ghost** [comment]
@yusefnapora 

Yusef owns libp2p docs, so I think he should think about this in the broader context of documentation, examples and tutorials.  I suggested he reach out to you @terichadbourne when he has some timel

**Comment by terichadbourne** [comment]
@mgoelzer Is the libp2p event originally scheduled for this June being pushed to 2020? Is there workshop content for that event that would make sense in ProtoSchool format?

**Comment by ghost** [comment]
@terichadbourne Yes, libp2p event being pushed indefinitely.  But we're in dire need of more online tutorial content, and I think Proto School could be a good venue to deliver that.

**Comment by terichadbourne** [comment]
@yusefnapora Any chance you'd have a little time this week to chat a bit about potential libp2p tutorial conent for ProtoSchool? Just want to make sure I'm incorporating any of your more immediate needs as I build out my roadmap and Q2 OKRs.

**Comment by yusefnapora** [comment]
Sure, thanks @terichadbourne!  Feel free to put something on my calendar anytime that works for you.
